sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -5742,6 +5742,22 @@ def test_pretty_Mod():
     assert upretty(2 * Mod(x, 7)) == ucode_str5
 
 
+def test_MatrixElement_printing():
+    # test case from issue #11821
+
+    ascii_str1 = "(-B + A)[0, 0]"
+    ucode_str1 = u("(-B + A)[0, 0]")
+
+    A = MatrixSymbol("A",1,3)
+    B = MatrixSymbol("B",1,3)
+    C = MatrixSymbol("C",1,3)
```

Comment:
As a minor detail, put a space after the comma, we are trying to model the code after the PEP-8 recommendations.